### PR TITLE
promo.kadokawa.co.jp

### PR DESCRIPTION
**Pull request by SKEIDs**
https://github.com/AdguardTeam/AdguardFilters/issues/103673

`promo.kadokawa.co.jp##.twitter`
`promo.kadokawa.co.jp##.sns`

<details><summary>Before</summary>

</details><br/>
<details><summary>After</summary>

</details><br/>

### CODE REVIEW

**Review by Yuki2718 — CHANGES_REQUESTED** [review]
Add `||platform.twitter.com/widgets.js$domain=promo.kadokawa.co.jp`

**Comment by SKEIDs** [comment]
add rule for twitter-timeline + widgets.js

**Comment by Yuki2718** [comment]
Off topic but I didn't realize my review comments are not visible to reviewee until I click "Submit review". Sorry @SKEIDs , it probably made some mis-communicaion btwn us. @zloyden Could you see my non-submitted (Pending) comments?

File: SocialFilter/sections/specific.txt [hunk 1]
```diff
@@ -39,10 +39,10 @@ filmyani.org,agrihaber04.com,afyonhaberturk.com,seffafbelediyecilik.com,denizli2
...
...forums.flyer.co.uk,fox38corpuschristi.com,fox47.com,foxkansas.com,foxrichmond.com,gazetaby.com,gercekizmir.com,gidavitrini.com.tr,govoritmoskva.ru,halklailiskiler.com.tr,hentairules.net,hizlifilmizlehd.com,hornygamer.com,inforesist.org,jeotermalhaberler.com,jetvis.ru,kachelmannwetter.com,karsolay.com,kbsi23.com,kdsm17.com,kevinpurcell.org,kg-portal.ru,kinopoisk.userecho.com,kp.ua,larende.com,losst.ru,magazinekibi.com,manisamanset.gen.tr,mforos.com,mogilev.online,muchohentai.com,multifilemirror.com,mvs.gov.ua,mytuner-radio.com,mywebtimes.com,namemc.com,news-journalonline.com,nk.org.ua,nkj.ru,noticierodigital.com,ntp-servers.net,obltv.ru,orbita.dn.ua,pctip.ru,podrobnosti.ua,ppt.ru,proglang.su,queenbet4.tv,raider.io,raleighcw.com,real-vin.com,rmj.ru,rusvesna.su,sanatatak.com,seirsanduk.us,simdifilmizle.com,sonkaleizmir.com,sp-mobile-blog.com,star64.tv,tayga.info,tomiks33.ru,torrentbe.net,torrentrg.org,tqm.com.ua,tribundergi.com,tva.in.ua,utofiles.com,washingtontimes.com,wcti12.com,womanel.com.ua,wpgh53.com,wtto21.com,wutv29.com,xn----8sbcm1bmcar0m.xn--p1ai,gunafrica.co.za,opentrackers.org,tv.ua,ewdn.com,meanjin.com.au,openworldnews.net,radiko.jp,promo.kadokawa.co.jp##.twitter-timeline
```
Comment (Yuki2718) [inline SocialFilter/sections/specific.txt:45]: In this case `promo.kadokawa.co.jp##.twitter-timeline` is not needed as covered by `promo.kadokawa.co.jp##.twitter`

**Review by Yuki2718 — CHANGES_REQUESTED** [review]

File: SocialFilter/sections/specific.txt (hunk 1 above)
Comment (SKEIDs) [reply SocialFilter/sections/specific.txt]: ok. removed

**Comment by zloyden** [comment]
@Yuki2718 I see only two of your comments.

File: SocialFilter/sections/specific.txt [hunk 2]
```diff
@@ -39,9 +39,9 @@ filmyani.org,agrihaber04.com,afyonhaberturk.com,seffafbelediyecilik.com,denizli2
...
...lab.ua|filmakinesi.net|filmold.com|forums.flyer.co.uk|fox38corpuschristi.com|fox47.com|foxkansas.com|foxrichmond.com|gercekizmir.com|gidavitrini.com.tr|govoritmoskva.ru|halklailiskiler.com.tr|hentairules.net|hizlifilmizlehd.com|hornygamer.com|inforesist.org|jeotermalhaberler.com|jetvis.ru|kachelmannwetter.com|karsolay.com|kbsi23.com|kdsm17.com|kevinpurcell.org|kg-portal.ru|kinopoisk.userecho.com|kp.ua|larende.com|losst.ru|magazinekibi.com|manisamanset.gen.tr|mforos.com|mogilev.online|muchohentai.com|multifilemirror.com|mvs.gov.ua|mytuner-radio.com|mywebtimes.com|namemc.com|news-journalonline.com|nk.org.ua|nkj.ru|noticierodigital.com|ntp-servers.net|obltv.ru|orbita.dn.ua|pctip.ru|podrobnosti.ua|ppt.ru|proglang.su|queenbet4.tv|raider.io|raleighcw.com|real-vin.com|rmj.ru|rusvesna.su|sanatatak.com|seirsanduk.us|simdifilmizle.com|sonkaleizmir.com|sp-mobile-blog.com|star64.tv|tayga.info|tomiks33.ru|torrentbe.net|torrentrg.org|tqm.com.ua|tva.in.ua|utofiles.com|washingtontimes.com|wcti12.com|womanel.com.ua|wpgh53.com|wtto21.com|wutv29.com|xn----8sbcm1bmcar0m.xn--p1ai|jansatta.com|gunafrica.co.za|opentrackers.org|tv.ua|ewdn.com|meanjin.com.au|openworldnews.net|radiko.jp|promo.kadokawa.co.jp
```
Comment (zloyden) [inline SocialFilter/sections/specific.txt:44]: Please move the domain to the beginning of the list.

**Review by zloyden — CHANGES_REQUESTED** [review]

**Review by zloyden — APPROVED** [review]

File: SocialFilter/sections/specific.txt (hunk 1 above)
Comment (Sergey-Lyapin) [reply SocialFilter/sections/specific.txt]: @Yuki2718 @SKEIDs Domain lists should be the same in these 2 rules, so I added the domain to `##.twitter-timeline` again

**Pull request merged**